tools/check-sonarqube.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# Copyright JS Foundation and other contributors, http://js.foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if [[ "${TRAVIS_REPO_SLUG}" == "jerryscript-project/jerryscript"
+  && ${TRAVIS_BRANCH} == "master"
+  && ${TRAVIS_EVENT_TYPE} == "push" ]]
+then
+  git fetch --unshallow
```

Comment:
Is this really necessary? I've found only one mention of `git fetch --unshallow` in SonarQube docs and that was in the description of the [Branch plugin](https://docs.sonarqube.org/display/PLUG/Branch+Plugin). Are we using that plugin or do we need this for some other reason?

Reply:
Yes it is. It was not included in the first patch, but I had to add it in a followup.

Reply:
See: #2325
There was warning in the analysis log.
>WARN: Shallow clone detected, no blame information will be provided. You can convert to non-shallow with 'git fetch --unshallow'.

Full log: https://travis-ci.org/jerryscript-project/jerryscript/jobs/377608402